docs/source/release/v6.1.0/framework.rst

```diff
@@ -24,6 +24,7 @@ Data Objects
 ------------
 
 - :ref:`Run <mantid.api.Run>` has been modified to allow multiple goniometers to be stored.
+- :ref:`FileFinder <mantid.api.FileFinderImpl>` has been modified to improve search times in some cases.
```

Comment:
Can we be a little more explicit in the *some* cases? Perhaps just say in the case of searching for multiple runs for the same instrument?